src/__stories__/header-story.tsx

```diff
@@ -29,7 +29,7 @@ export const Default: StoryComponent = () => {
         true
     );
     const [withBreadcrumbs, breadcrumbsCheckbox] = useCheckbox('With breadcrumbs (desktop only)', true);
-    const [bleed, bleedCheckbox] = useCheckbox('Bleed', false);
+    const [bleed, bleedCheckbox] = useCheckbox('Bleed', true);
```

Comment:
Instead of changing the default value to true, you should check the checkbox in the bleed test.

The previous HeaderLayout screenshot tests should not change